### DISCUSSION TITLE
Adding a map stats in mod customization menu

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#24694] [Ideas]
Yes, I know it's duplicate of #18189 and #18172, but I'm going to actually discuss implementation of this thing.

During the mod customization (during mod select it will also be visible) - there will be this overlay.
![image](https://github.com/ppy/osu/assets/89256026/e939361c-a7f1-4d59-96a5-45da0b1c83b8)
It will display the actual stats of the map. I've selected the ones that players usually care about - star rating, BPM and difficulty attributes.

Using this you can very easily calibrate DT rate adjust and DA attributes adjust, so the map will have exactly AR10.3 and 7* for example

The difficulty adjust will show raw AR/OD values when this overlay will show them after all mods applied

**Comment by monochrome22** [comment]
I don't know how important compatibility with UI scaling is considered, but this wouldn't work with max UI scaling.
If UI scaling is important, then a better place would be above the buttons (Back, Mod customisation, Deselect All).

**Reply by Givikap120** [reply]
This is works fine for mod customization menu, but it won't work with mod select menu
I can do like this old layout
https://www.figma.com/file/0DrBgpMa6rJ4DvruF5pv2S/Overlay-Animation?type=design&node-id=0-1&mode=design
But it will be the same problem - not enough space with max UI scaling

**Reply by monochrome22** [reply]
I didn't notice you wanted to add it to mod select as well. Above the buttons won't work then.
Personally I like the figma design, but making it work with UI scaling would require also adjusting other elements.

**Comment by bdach** [comment]
https://github.com/ppy/osu/pull/19929#issuecomment-1230730804 is the last time this was tried. tl;dr: design bad, pr got stuck in review hell.

what you have there _could_ be okay, but on discord you said something about adding that to song select footer too. i disagree with that. that should not be required. the post-mod info should show on the beatmap wedge / places where that info normally goes.

**Reply by monochrome22** [reply]
He said mod select.

**Comment by Givikap120** [comment]
Okay, this is my developed version
Design advice is welcome

https://github.com/ppy/osu/assets/89256026/365d33e2-3f91-45cb-81af-7f0cfed47424

**Reply by peppy** [reply]
I think this looks great, apart from the incorrect spacing between the last button and your control.
